Draft for PR description (#220)
cosmo-epfl/librascal: docs/source/dev_guide/how_to_contribute.rst

      }
  }

> Once that the method that you need as been coded, you can save the file(s) and proceed. Note that adding a ``.cc`` file requires its registration in the folder's CMakeLists.txt so that it can be compiled.

Reviewer: ```suggestion
Once that the method that you need as been implemented, you can save the file(s) and proceed. Note that adding a ``.cc`` file requires its registration in the folder's CMakeLists.txt for it to be compiled.
```

> Once that the method that you need as been implemented, you can save the file(s) and proceed. Note that adding a ``.cc`` file requires its registration in the folder's CMakeLists.txt for it to be compiled.

Reviewer 2: ```suggestion
Once the method that you need has been implemented, you can save the file(s) and proceed. Note that adding a ``.cc`` file requires its registration in the folder's CMakeLists.txt for it to be compiled.
```
